src/test/scala/org/broadinstitute/hail/methods/ExprSuite.scala

```diff
@@ -12,28 +12,28 @@ import org.testng.annotations.Test
 class ExprSuite extends SparkSuite {
 
   @Test def exprTest() {
-    val symTab = Map("i" ->(0, TInt),
-      "j" ->(1, TInt),
-      "d" ->(2, TDouble),
-      "d2" ->(3, TDouble),
-      "s" ->(4, TString),
-      "s2" ->(5, TString),
-      "a" ->(6, TArray(TInt)),
-      "m" ->(7, TInt),
-      "as" ->(8, TArray(TStruct(("a", TInt),
+    val symTab = Map("i" -> (0, TInt),
```

Comment:
How did you set your Intellij format settings to do this?  It is a nice thing and I want it.